### PR TITLE
feat(dedicated.network-security): update security dashboard

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-503
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

- retrieve only IP v4 from selected service
- reset selected IP when service is reset
- check IP set is valid

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js [hunk 1]
```diff
@@ -99,27 +110,46 @@ export default class ScrubbingCenterController extends AbstractCursorDatagridCon
...
+    this.isValid = isValid;
+    if (isValid) {
+      this.selectedIp = value;
+      if (value.indexOf('/') === -1) {
+        if (value.indexOf('.') > -1) {
+          this.selectedIp = `${value}/32`;
+        } else if (value.indexOf(':') > -1) {
+          this.selectedIp = `${value}/128`;
+        }
+        this.model = this.selectedIp;
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:142]:
```suggestion
      if (!value.includes('/')) {
        if (value.includes('.')) {
          this.selectedIp = `${value}/32`;
        } else if (value.includes(':')) {
          this.selectedIp = `${value}/128`;
        }
        this.model = this.selectedIp;
```

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.html [hunk 2]
```diff
@@ -58,15 +61,19 @@
...
     </div>
 
-    <div class="text-center" data-ng-if="$ctrl.isLoading">
-        <oui-spinner></oui-spinner>
-    </div>
+    <div
+        data-ng-if="$ctrl.isEmpty"
+        class="text-center m-3"
+        data-translate="network_security_dashboard_empty_table"
+    ></div>
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.html:68]:
```suggestion
    <p
        data-ng-if="$ctrl.isEmpty"
        class="text-center m-3"
        data-translate="network_security_dashboard_empty_table"
    ></p>
```

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.html [hunk 3]
```diff
@@ -40,6 +40,9 @@
                     data-placeholder="{{:: 'network_security_dashboard_select_search_ip' | translate}}"
                     data-ng-if="!$ctrl.isServiceSelected"
                 ></oui-search>
+                <div class="oui-color-ae-500" data-ng-if="!$ctrl.isValid">
+                    {{ 'network_security_dashboard_invalid_ip' | translate}}
+                </div>
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.html:45]:
```suggestion
<p class="oui-color-ae-500" data-ng-if="!$ctrl.isValid" data-translate="network_security_dashboard_invalid_ip"></p>
```

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
+import isNaN from 'lodash/isNaN';
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:1]: Why not using es6 isNaN() instead ?

File: packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import isNaN from 'lodash/isNaN';
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js:1]: same as https://github.com/ovh/manager/pull/10489#discussion_r1427761489

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js (hunk 4 above)
Comment (Steffy29) [reply packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:1]: The habit

File: packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js (hunk 5 above)
Comment (Steffy29) [reply packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js:1]: Same

**Review by qpavy — APPROVED** [review]

**Pull request merged**